### PR TITLE
Add option to auto create IAM policy on application account

### PR DESCRIPTION
**Pull request by radepal**

### CODE REVIEW

File: application/eu-central-1/iam/auto/variables.tf [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  description = "AWS application account number (without hyphens)"
+}
+
+variable "k8s_cluster_name_postfix" {
+  description = "Domain name of Kubernetes cluster (currently only k8s.local is supported)"
+  default     = "k8s.local"
+}
+
+variable "logs_not_resource" {
+  description = "List of resources that log police will NotResource, empty least mean that Resource * is set"
```
Comment (mzawadzk) [inline application/eu-central-1/iam/auto/variables.tf]: List of resources that log police will use for NotResource, empty means that Resource * is set

File: application/eu-central-1/iam/auto/variables.tf [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  default     = []
+  type        = "list"
+}
+
+variable "auto_IAM_mode" {
+  description = "Create IAM Policies in AWS (default false)"
```
Comment (mzawadzk) [inline application/eu-central-1/iam/auto/variables.tf]: We should not put defaults into description.

File: application/eu-central-1/iam/auto/variables.tf (hunk 1 above)
Comment (mietlinski-nc) [reply application/eu-central-1/iam/auto/variables.tf]: +1

File: application/eu-central-1/iam/manual/main.tf [hunk 3]
```diff
@@ -0,0 +1,16 @@
+module "application_iam" {
+  source = "github.com/kentrikos/terraform-aws-iam-generator//application"
+
+  region              = "${var.region}"
+  product_domain_name = "${var.product_domain_name}"
+  environment_type    = "${var.environment_type}"
+
+  operation_aws_account_number   = "${var.operation_aws_account_number}"
```
Comment (mzawadzk) [inline application/eu-central-1/iam/manual/main.tf]: operations

**Pull request merged**